.github/workflows/zips.yml

```
@@ -0,0 +1,122 @@
+name: Plugin Zip Files
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+    tags: ['**']
+  pull_request:
+    types:
+      - opened
+      - closed
+      - reopened
+      - synchronize
+
+jobs:
+  build-zips:
+    name: Build Zips
+    runs-on: ubuntu-latest
+    if: ${{ ! ( ( github.event_name == 'pull_request' && github.event.action == 'closed' ) || github.event.pull_request.head.repo.fork ) }}
```

Same here, let's remove the expression syntax.
```suggestion
    if: ! ( ( github.event_name == 'pull_request' && github.event.action == 'closed' ) || github.event.pull_request.head.repo.fork )
```